code/game/objects/items/devices/autopsy.dm

```diff
@@ -7,16 +7,24 @@
 	w_class = WEIGHT_CLASS_TINY
 	origin_tech = "materials=1;biotech=1"
 	var/list/datum/autopsy_data_scanner/wdata = list()
-	var/list/datum/autopsy_data_scanner/chemtraces = list()
```

Comment:
you can't typecast lists, but I can respect doing so in this manner to denote what is even stored in them....but in this case, it's literally a text string and a number, not even a datum.